src/views/Hearing/index.js

```diff
@@ -225,12 +226,14 @@ function groupSections(sections) {
     const sectionGroup = find(sectionGroups, group => section.type === group.type);
     if (sectionGroup) {
       sectionGroup.sections.push(section);
+      sectionGroup.n_comments += section.n_comments;
```

Comment:
This is not mutating passed in data. The section group is created in the very same function, by iterating through the sections that have been passed.